homeassistant/components/sensor/thethingsnetwork_storage.py

```diff
@@ -0,0 +1,164 @@
+"""
```

Comment:
let's name this platform `thethingsnetwork`. I would expect eventually we would use a combination of `storage` to get the current value and then `mqtt` to be notified of new values.